content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -269,7 +269,7 @@ is in the `Pending` state but should have a condition `Initialized` set to false
 If the Pod [restarts](#pod-restart-reasons), or is restarted, all init containers
 must execute again.
 
-Changes to the init container spec are limited to the container image field.
+Changes to the init container spec are limited to the init container image field (.spec.initContainers.image).
```

Comment:
`.spec.initContainers` is a list so `.spec.initContainers.image` is not the correct field path.

```suggestion
Changes to the init container spec are limited to the `image` field. If you try to change
any other field within `.spec.initContainers`, the API server rejects the attempt.
```

Reply:
Updated the current PR, please take a review.

Reply:
@utkarsh-singh1 I think you still have `.spec.initContainers.image`

Reply:
Updated above changes, @marquiz